package/gluon-web-private-wifi/luasrc/lib/gluon/config-mode/model/admin/privatewifi.lua

```diff
@@ -38,12 +38,12 @@ warning:depends(enabled, true)
 local ssid = s:option(Value, "ssid", translate("Name (SSID)"))
 ssid:depends(enabled, true)
 ssid.datatype = "maxlength(32)"
-ssid.default = uci:get('wireless', primary_iface, "ssid")
```

Comment:
There are remaining uses of `primary_iface` in this file that must be replaced as well.